python/requirements.txt

```diff
@@ -17,6 +7,7 @@ pandas==1.2.1
 pytest==6.2.1
 pytest-cov==2.11.0
 pytest-html==3.1.1
+PyYAML==5.4.1
```

Comment:
@Ullimague @jakemannix I had to add this because the library that was installing PyYAML transitively must have been moved to `build-requirements.txt` now.

Reply:
hmmm... but do we have approval for it?